Tests/test_enum.py

```diff
@@ -0,0 +1,56 @@
+#####################################################################################
+#
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# This source code is subject to terms and conditions of the Apache License, Version 2.0. A
+# copy of the license can be found in the License.html file at the root of this distribution. If
+# you cannot locate the  Apache License, Version 2.0, please send an email to
+# ironpy@microsoft.com. By using this source code in any fashion, you are agreeing to be bound
+# by the terms of the Apache License, Version 2.0.
+#
+# You must not remove this notice, or any other, from this software.
+#
+#
+#####################################################################################
+
+import clr
+clr.AddReference('IronPython')
+from IronPython.Compiler.Ast import Arg
+
+Arg(None)
+
+import sys
+
+from iptest import IronPythonTestCase, is_cli, run_test, skipUnlessIronPython
```

Comment:
I don't see how the code above relates to the enum tests? You can probably get rid of everything before this line and use the .NET foundation copyright.
```
# Licensed to the .NET Foundation under one or more agreements.
# The .NET Foundation licenses this file to you under the Apache 2.0 License.
# See the LICENSE file in the project root for more information.
```